systemtest/src/main/java/io/strimzi/systemtest/utils/kubeUtils/objects/PodUtils.java

```diff
@@ -56,6 +57,10 @@ public static String getInitContainerImageName(String podName) {
     }
 
     public static void waitForPodsReady(LabelSelector selector, int expectPods, boolean containers) {
+        waitForPodsReady(selector, expectPods, containers, () -> { });
```

Comment:
What does it mean `() -> { }` ?

Reply:
This is used in case you don't want to onTimeout code execution - nothing will happen, just empty lambda - empty runnable - if you have any idea how to do it better - I'm open for any ideas. I think this is good if we don't want to display anything in tests after timeout exception is thrown.